controllers/base_controller.js

```diff
@@ -67,6 +71,17 @@ module.exports = function BaseControllerModule(pb) {
      * @type {String}
      */
     var ALERT_PATTERN = '<div class="alert %s error_success">%s<button type="button" class="close" data-dismiss="alert" aria-hidden="true">&times;</button></div>';
+
+    /**
+     * The prefix in the content-type header that indicates the charset used in 
+     * the encoding
+     * @static
+     * @private
+     * @readonly
+     * @property CHARSET_HEADER_PREFIX
+     * @type {String}
+     */
+    var CHARSET_HEADER_PREFIX = 'charset=';
```

Comment:
This has been removed.

From: Brian <notifications@github.com<mailto:notifications@github.com>>
Reply-To: pencilblue/pencilblue <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Thursday, July 9, 2015 at 5:24 PM
To: pencilblue/pencilblue <pencilblue@noreply.github.com<mailto:pencilblue@noreply.github.com>>
Cc: Ben Tidwell <ben.tidwell@careerbuilder.com<mailto:ben.tidwell@careerbuilder.com>>
Subject: Re: [pencilblue] Multi Tenancy (#680)


In controllers/base_controller.js<https://github.com/pencilblue/pencilblue/pull/680#discussion_r34305508>:

> @@ -67,6 +71,17 @@ module.exports = function BaseControllerModule(pb) {
>       * @type {String}
>       */
>      var ALERT_PATTERN = '<div class="alert %s error_success">%s<button type="button" class="close" data-dismiss="alert" aria-hidden="true">&times;</button></div>';
> +
> +    /**
> +     * The prefix in the content-type header that indicates the charset used in
> +     * the encoding
> +     * @static
> +     * @private
> +     * @readonly
> +     * @property CHARSET_HEADER_PREFIX
> +     * @type {String}
> +     */
> +    var CHARSET_HEADER_PREFIX = 'charset=';


this variable isn't used anywhere in the file and is private. it should be removed.

-
Reply to this email directly or view it on GitHub<https://github.com/pencilblue/pencilblue/pull/680/files#r34305508>.